carsus/io/cmfgen/base.py

```diff
@@ -698,10 +735,48 @@ def _get_levels_lines(self, data):
             lns = lns.reset_index()
             lns_list.append(lns)
 
+            if 'ionization_energy' in reader.keys():
+                ioz_list.append({'atomic_number': ion[0], 
+                                 'ion_charge': ion[1], 
+                                 'ionization_energy': reader['ionization_energy']})
+
+            if 'cross_sections' in reader.keys():
+                if ion == (1,0):
+                    
+                    # Extracted from the header of HYD files since the `data` object 
+                    # passed to this method does not contain header information.
+
+                    n_levels = 30
+                    l_points, l_start_u, l_del_u = 97, 0.0, 0.041392685
+                    n_points, n_start_u, n_del_u = 145, 0.0, 0.041392685
+
+                    hyd = reader['hyd'].apply(lambda x: 10 ** (8 + x))
+                    gbf = reader['gbf']
+
+                    hyd_phixs, hyd_phixs_energy_grid_ryd = {}, {}
+                    hyd_gaunt_factor, hyd_gaunt_energy_grid_ryd = {}, {}
+
+                    for n in range(1, n_levels+1):
+
+                        lambda_angstrom = lvl.loc[n-1, 'Lam(A)']
+                        e_threshold_ev = HC_IN_EV_ANGSTROM / lambda_angstrom
+
+                        hyd_gaunt_energy_grid_ryd[n] = [e_threshold_ev / RYD_TO_EV * 10 ** (n_start_u + n_del_u * i) for i in range(n_points)]
+                        hyd_gaunt_factor[n] = gbf.loc[n].tolist()
+
+                        for l in range(0, n):
+                            hyd_phixs_energy_grid_ryd[(n, l)] = [e_threshold_ev / RYD_TO_EV * 10 ** (l_start_u + l_del_u * i) for i in range(l_points)]
+                            hyd_phixs[(n,l)] = hyd.loc[(n,l)].tolist()
+
+                pxs = self.cross_sections_squeeze(reader['cross_sections'][0], lvl, hyd_phixs_energy_grid_ryd, hyd_phixs, hyd_gaunt_energy_grid_ryd, hyd_gaunt_factor)
+                pxs['atomic_number'] = ion[0]
+                pxs['ion_charge'] = ion[1]
```

Comment:
```suggestion
                pxs['ion_number'] = ion[1]
```
To be compatible with TARDIS.